domain/src/main/java/com/teamagam/gimelgimel/domain/layers/LoadAllCachedLayersInteractor.java

```diff
@@ -23,16 +23,19 @@ public LoadAllCachedLayersInteractor(ThreadExecutor threadExecutor,
   }
 
   @Override
-  protected Iterable<SubscriptionRequest> buildSubscriptionRequests(
-      DataSubscriptionRequest.SubscriptionRequestFactory factory) {
+  protected Iterable<SubscriptionRequest> buildSubscriptionRequests(DataSubscriptionRequest.SubscriptionRequestFactory factory) {
     DataSubscriptionRequest<?> request = factory.create(Observable.just(null),
         observable -> observable.flatMapIterable(x -> mLayersLocalCache.getAllCachedLayers())
             .map(this::recreateAsUnimportant)
-            .doOnNext(vl -> mProcessNewVectorLayerInteractorFactory.create(vl, null).execute()));
+            .doOnNext(vl -> mProcessNewVectorLayerInteractorFactory.create(vl).execute()));
     return Collections.singletonList(request);
   }
 
   private VectorLayer recreateAsUnimportant(VectorLayer vectorLayer) {
-    return VectorLayer.copyWithDifferentSeverity(vectorLayer, VectorLayer.Severity.REGULAR);
+    VectorLayer newVectorLayer =
+        VectorLayer.copyWithDifferentSeverity(vectorLayer, VectorLayer.Severity.REGULAR);
+    newVectorLayer.setURl(null);
```

Comment:
Yeah, the interactor that processing it, depends on null URL (to get the cached file).